com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/AbsoluteNode.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class AbsoluteNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,          // Version
+            "Absolute",    // Name
+            "Out = abs(In);",
+            new ParameterDescriptor("In", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("Out", TYPE.Vector, Usage.Out)
+        );
+
+        public static Dictionary<string, string> UIStrings => new()
+        {
+            { "Name.Synonyms", "positive" },
+            { "Tooltip", "returns the positive version of the input " },
+            { "Parameters.In.Tooltip", "input value" },
```

Comment:
We should get together and standardize on this one.  For all of the inputs/outputs where there wasn't really a valuable description, I think I just left the tooltip out. But let's decide on a standard for these so that our style's match.

Reply:
I agree that some of the tooltips are a bit unnecessary. The reason that I still keep them is that I feel when the users get feedback hovering over the output port they might also expect the input ports to have it as well? Some of the nodes have tooltips for both of the ports but some don't just kinda make me feel uneven🤔 What do you think?

Reply:
My understanding was the even if we didn't provide an explicit tooltip, there would still be a tooltip that tells the user the data type of the port.  If that's not correct and the user gets nothing when they mouse over ports with no tooltip string, that wouldn't be good.  I'll go ahead and update mine so that all ports have them - just in case.

Reply:
Sounds good!